shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -446,7 +446,11 @@ namespace datadog::shared::nativeloader
 
                     moduleName = moduleName + EndLWStr;
                     instrumented_assembly_generator::WriteTextToFile(instrumented_assembly_generator::ModulesFileName, moduleName);
-                    instrumented_assembly_generator::CopyOriginalModuleForInstrumentationVerification(modulePath);
+                    if (instrumented_assembly_generator::IsCopyingOriginalsModulesEnabled())
+                    {
+                        // by default this is disabled but we keep that option to be able to do the instrumentation verification on a offline environment
```

Comment:
```suggestion
                        // This option means we copy every loaded assembly to the InstrumentationVerification folder. It is off by default,
                        // but can be useful as when we are working on an escalation,
                        // we can request that the customer turns it on, and then the customer
                        // will only need to upload to the contents of the InstrumentationVerification folder to us for offline analysis, 
                        // so we won't have to ask the customer to go and locate wherever it was that the assembly was loaded from at runtime.
```

Reply:
Fix in d6e34e581c70cc2271806c12db2c3d4f2caf78f1